commands/Alarm.go

```diff
@@ -35,7 +36,7 @@ func NewCmdAlarm(data []byte) *CmdAlarm {
 	return alarm
 }
 
-func (self *CmdAlarm) String() string {
+func (self *AlarmReport) String() string {
```

Comment:
don't name receivers "self". That isn't Go style.

See https://github.com/golang/go/wiki/CodeReviewComments#receiver-names